### ISSUE TITLE
Add HTML examples to WCAG 2.5.3 label-in-name.html#examples

### ISSUE THREAD

**Issue by ljoakley** [w3c/wcag#4994]
To help people understand how labels and names actually work, add HTML examples to both Examples and Failures. As an Example: ```<button aria-label=OK>OK</button>``` and a failure ```<button aria-label=Submit>OK</button>```.

**Comment by patrickhlauke** [comment]
wouldn't that be better as a technique, to keep the understanding more tech-agnostic?

**Comment by detlevhfischer** [comment]
There is mention of button text both in the listing under "Identifying label text for components" and later in an example under punctuation, but it is true there is more emphasis on elements with adjacent labels, like text fields or inputs type checkbox or radio.

What I feel is missing is links. These only apppear in the initial phrase "When these match, speech-input users (i.e., users of speech recognition applications) can navigate by speaking the visible text labels of components, such as menus, **links,** and buttons, that appear on the screen".

Is it crystal-clear that links are also covered? I think they should appear separately in the list in section "Identifying label text for components" to make that crystal clear. I can prepare a PR to add such a list item if no one thinks I got it wrong here.

A related issue concerns [F96](https://www.w3.org/WAI/WCAG22/Techniques/failures/F96 ): the unclear requirements regarding the accessible name of images of text. I will create a separate issue for that.
